README.md

```diff
@@ -0,0 +1,14 @@
+# Contiv Prometheus Integration for Kubernetes
+
+## Getting Started
+
+In order to run this exporter for your cluster, please follow [this]() tutorial.
+
+## Exporter Architecture
+
+![Exporter Pipeline](exporter.png)
+
```

Comment:
Can you split this up into sections:

1. Prometheus setup - configuration - yaml file that is volume mounted. Add a link to the file here and describe how the different sections allow it to do pod discovery.
2. Netplugin/Netmaster exporters - How the exporter is a web service providing a /metrics rest end point at different ports for master/netplugin
How the pods themselves are configured with annotations to allow for the data to be fetched.
3. Exporter implementation - Ruby code details
4. OVS details - how there is a server started by netplugin on the port that you access and how it has traffic stats
4. Data extraction pipeline - prometheus to exporter to netmaster/ovs pipeline